src/payments/providers/stripe/stripe.provider.ts

```
@@ -281,6 +281,20 @@ export class StripeProvider implements PaymentProvider {
         metadata = session.metadata || {};
         break;
 
+      case 'checkout.session.expired': {
```

ESLint

[error] Replace `'checkout.session.expired'` with `"checkout.session.expired"`

(prettier/prettier)